.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,65 @@
+name: 🐞 Bug
+description: File a bug/issue
+title: "[Bug] <title>"
+labels: "bug"
+
+body:
+- type: checkboxes
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+    - label: I have searched the existing issues.
+      required: true
+      
```

Comment:
@designe suggested ```Would you like to work on the issue``` attribute at #9.
- [ ] Would you like to work on the issue?

If this attribute is checked, we can assign this issue to person who opened this issue.
If this attribute is not checked, we can assign this issue to other who is interested in this issue.
So, would you add this attribute? :smile:

Reply:
@yeonns I like this attribute ;)